docs/yaml/yaml.go

```diff
@@ -206,6 +211,9 @@ func genFlagResult(flags *pflag.FlagSet) []cmdOption {
 		if _, ok := flag.Annotations["swarm"]; ok {
 			opt.Swarm = true
 		}
+		if os, ok := flag.Annotations["ostype"]; ok && len(opt.OSType) == 0 && len(os) > 0 {
+			opt.OSType = os[0]
```

Comment:
for the record; I decided to only use the first ostype here; theoretically, the annotation could have multiple, but we don't use that (and possibly never will); making this a single value is to make it consistent with the ostype on commands, and will make it easier to use for the docs team

@vdemeester let me know if you want me to add that as a comment or commit message

Reply:
A comment here or in the commit message seems safe(r) :angel: :stuck_out_tongue:

Reply:
Let me do both